### ISSUE TITLE
Align narratives with D4D

### ISSUE THREAD

**Issue by samuele-mattiuzzo** [IATI/IATI-Rulesets#231]

**Comment by stevieflow** [comment]
> Some narratives have been clarified on D4D's Ruleset repository. 

Just so I understand this correctly - is it therefore safe to assume that some of the rules have *changed* during the implementation of the validator?  

It doesn't matter if there is a belief that there is no semantic change - we need to see such clarifications before any central documentation has the suggestion of being updated

PR such as https://github.com/IATI/IATI-Rulesets/pull/98 also seem to suggest some changes to the rulesets

This is quite concerning, given that the rulesets are within the SSOT and subject to strict change control

**Comment by samuele-mattiuzzo** [comment]
No, it's not safe to assume they have CHANGED. The definitions have ben CLARIFIED, as in made a bit more understandable.

**Comment by stevieflow** [comment]
Appreciated @samuele-mattiuzzo - but I just don't feel comfortable with assumptions.  

Are the changes / diffs that came from the clarifications written down?

**Comment by stevieflow** [comment]
Any news @samuele-mattiuzzo ?

**Comment by akmiller01** [comment]
Keeping this open, but marking as "standard management" as it seems like there may be a question about the proper provenance of some rules.
